packages/ui/lib/components/Form/FieldLayout.tsx

```diff
@@ -37,13 +38,18 @@ export function FieldLayout(props: Props) {
     append,
     required,
     optional,
+    hidden,
   } = props
   const labelSizeStyle = SIZE_STYLES[size!]
   const labelClass = twMerge('px-1', labelSizeStyle)
   const descriptionClass = twMerge('text-gray-600', TEXT_SIZE[size])
   const errorClass = twMerge('text-red-500', TEXT_SIZE[size])
   const successClass = twMerge('text-green-500', TEXT_SIZE[size])
 
+  if (hidden) {
+    return null
+  }
```

Comment:
I'm confused why both this and the hidden type. Hidden type should make input invisible already, no?

Reply:
yes but the label and such are still rendered... hence this